kafka/src/main/java/org/axonframework/extensions/kafka/eventhandling/consumer/OffsetCommitType.java

```diff
@@ -0,0 +1,10 @@
+package org.axonframework.extensions.kafka.eventhandling.consumer;
+
+/**
+ * Enum to define how the consumer will handle committing offsets.
+ */
+public enum OffsetCommitType {
+    AUTO,
+    COMMIT_ASYNC,
+    COMMIT_SYNC
```

Comment:
Nit: Each enumeration option should have some JavaDoc! Would you mind adding that, @bradskuse?